src/themis/secure_comparator.c

```diff
@@ -38,7 +38,7 @@ static themis_status_t ed_sign(uint8_t pos, const uint8_t *scalar, uint8_t *sign
 {
 	uint8_t r[ED25519_GE_LENGTH];
 	ge_p3 R;
-	uint8_t k[64];
+	uint8_t k[64]; //temp buffer for group elements and hash
```

Comment:
it's concern simple short functions. i don't think that another name for local variable will so confuse readers. if some variables are global and used in many pieces of code then it's really can confuse

anyway, what the problem to use good variable names at start and use them in papers too. not good to do bad at start and depend on bad choice in future. if one famous man use shitty variable names it's not mean that all need to do this too